test_runner/fixtures/neon_fixtures.py

```diff
@@ -2553,6 +2553,30 @@ def tenant_describe(self, tenant_id: TenantId):
         response.raise_for_status()
         return response.json()
 
+    def nodes(self):
+        """
+        :return: list of {"id": ""}
+        """
+        response = self.request(
+            "GET",
+            f"{self.api}/control/v1/node",
+            headers=self.headers(TokenScope.ADMIN),
+        )
+        response.raise_for_status()
+        return response.json()
+
+    def node_attached(self, node_id: NodeId):
```

Comment:
"attached" means nodes which are fully attached (excluding secondaries).  Let's tweak this name to something like "node_shards" if it should include all secondaries as well.